### PR TITLE
Update CoreFx to preview2-25405-01

### PR DESCRIPTION
**Pull request by ericstj**
/cc @eerhardt @dsplaisted @nguerrera 

Doing this manually.  We need to get maestro hooked up for this, if that' is how you typically update these.

### CODE REVIEW

**Review by eerhardt — APPROVED** [review]
yes, Maestro is how this is normally handled.  dotnet/sdk doesn't support auto-update stuff at all - either for producing or consuming.

File: build/DependencyVersions.props [hunk 1]
```diff
@@ -13,7 +13,7 @@
     <NuGetVersion>4.3.0-preview2-4095</NuGetVersion>
     <NewtonsoftJsonVersion>9.0.1</NewtonsoftJsonVersion>
     <SystemReflectionMetadataVersion>1.4.2</SystemReflectionMetadataVersion>
-    <NETStandardLibraryNETFrameworkVersion>2.0.0-preview2-25331-02</NETStandardLibraryNETFrameworkVersion>
+    <NETStandardLibraryNETFrameworkVersion>2.0.0-preview2-25405-01</NETStandardLibraryNETFrameworkVersion>
```
Comment (eerhardt) [inline build/DependencyVersions.props]: QUESTION: This version seems later than our current "escrow" version of Microsoft.NETCore.App `2.0.0-preview2-25402-02`.  Is that a problem?

Comment (ericstj) [inline build/DependencyVersions.props]: I just took the latest build out of `release/2.0.0`, I can choose a different version if one is more appropriate.  Let me ask around.

Comment (ericstj) [inline build/DependencyVersions.props]: I got confirmation from @Petermarcu that the version that was just picked up by core-setup is the correct one: https://github.com/dotnet/core-setup/commit/1768bf1d4c7fd317d4a99d6c0cfba851ff2a61cd, so it is 25405-01 as I am using here.

**Comment by ericstj** [comment]
@livarcocc Do I need any other approval to merge this codeflow PR?

**Pull request merged**